examples/basic/main.go

```diff
@@ -0,0 +1,170 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/momentohq/client-sdk-go/auth"
+	"github.com/momentohq/client-sdk-go/config"
+	"github.com/momentohq/client-sdk-go/momento"
+	momento_redis "github.com/momentohq/momento-go-redis-client/momento-redis"
```

Comment:
Super nit-picky, but golang doesn't like snake case. Please rename to `momentoredis` or `momentoRedis`.